site/content@beta/backend-apis/core-info.md

```diff
@@ -0,0 +1,155 @@
+
+5. Customers can send messages even when agents are **offline**.
+
+6. **Multiple users** (Agents or Customers) can participate in a single chat. 
+
+7. Every user can have **multiple concurrent chats**. Read how it applies to [Agents](#chatting-as-an-agent) and [Customers](#chatting-as-a-customer).
+<!-- Agent moze miec nieskoncozna liczbe czatow. W kazdym czacie moze byc tylko 1 agent (ci co supervisuja nie moga uczestniczyc w czacie, tylko moga komentarze wysylac) -->
+
+8. The algorithm, which decides how chats are distributed between Agents is called **routing**. It's documented in the [Routing](#routing) section.
+
+
+### The flow 
+
+To better understand the flow between services, watch a short video.
+
+<!-- Animacja do zmiany. Zajmie sie tym Gosia oraz Bartek. -->
+
+<video loop width="750" height="500" style="height: auto;" controls>
+<source type="video/mp4" src="/beta-docs/platform-overview/images/simple_event_schema.mp4">
+</video>
+
+
+## Chat routing
+
+Routing is the process of **assigning chats to agents**. The primary goal of the routing mechanism is to distribute chats to all available agents who use the same license.  
+<!-- In the future, it will be possible to create a custom routing mechanism and run it on the LiveChat platform. -->
+
+Chats are being assigned to agents **either automatically or manually**. Our default routing mechanisms are described in [Understanding chat routing](https://www.livechatinc.com/help/understanding-chat-routing/).
+
+<!-- ### Automatic routing
+
+The diagram below present the flow of automatic routing.
+
+![Automatic routing](automatic-routing.png "automatic routing")
+
+### Manual routing
+
+The diagram below present the flow of manual routing.
+
+![Manual routing](manual-routing.png "manual routing") -->
+
+<!-- 1. Czy dajemy te grafiki, przerabiamy je? 
+
+    2. Czy wspominamy o custom routingu, ktory bedzie mozliwy w przyszlosci? 
+    
+    3. Czy rezygnujemy z opisow i linkujemy do artykulu w Help Center? 
+    
+    4. Fajnie by bylo tu kiedys nagrac filmik z lektorem  -->
+
+### Router system messages
+
+While the chat changes its state, the router sends system messages. Read more about [system messages](#system-message).
+
+<!-- W tym zdaniu chodzi o to, ze routing zmienia stan chatow? Przerobilam tak jak mi sie wydaje -->
+
+####  Overview
+
+| Message text | System message type |
+|--------------|---------------------|
+| `Chat is unassigned because <agent_name> hasn't replied in <minutes_number> minutes` | `routing.unassigned` |
+| `Chat assigned to <agent_name> because <agent_name> hasn't replied in <minutes_number> minutes` | `routing.assigned` |
+| `Chat archived due to long inactivity` | `routing.archived` |
+| `Chat is idle due to <minutes_number> minutes of inactivity` | `routing.idle` |
+| `Chat archived because customer was banned by <agent> for N days` | `customer_banned` |
+
+#### System messages: routing.assigned
```

Comment:
Nie jestem pewny ale to nie są system message tylko propertiesy chat/threada/eventu

Reply:
Tu jest lista wszystkich system message'y jak byś chciała :D
https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/497254461/LC3+system+messages

Reply:
section is removed from this doc anyway